docs/observability/sdo/set-up-sdo/sdo-manual-configuration.md

```diff
@@ -102,6 +102,8 @@ If you're using CircleCI for Build and Deploy, do the following:
 1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  In the main Sumo Logic menu, select **Manage Data > Logs > Field Extraction Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Logs** select **Field Extraction Rules**. You can also click the **Go To...** menu at the top of the screen and select **Field Extraction Rules**.  Kanso-->
 1. [Create two FERs](/docs/manage/field-extractions/create-field-extraction-rule.md) for CircleCI build and deploy events. The parse expression and the scope of the FERs is available [here](https://raw.githubusercontent.com/SumoLogic/sumologic-solution-templates/master/software-development-optimization-terraform/sdo_app_artifacts/sdo_fer.txt).
 
+1. In order to use uncertified orbs (partner or community), your organization’s administrator must opt-in to allow uncertified orb usage on the **Organization Settings > Security** page for your org.
```

Comment:
@himanshu219, I couldn't find **Organization Settings > Security** in Sumo Logic. Is it in CircleCI?

Reply:
yes 
<img width="1433" alt="Screenshot 2024-09-11 at 12 01 23 PM" src="https://github.com/user-attachments/assets/fcffc1ab-3ad1-4cc1-ad35-6708aacd2930">

I also noticed now that its unclear since rest of the steps are in sumo.

Reply:
@jpipkin1 I discussed with Himanshu and we changed the flow of the doc accordingly. Let me know your feedback on the same.

Reply:
It looks good, @himanshu219.